runner/database_neo4j.go

```diff
@@ -0,0 +1,57 @@
+package runner
+
+import (
+	"io"
+
+	jsonutil "github.com/multiprocessio/go-json"
+	"github.com/neo4j/neo4j-go-driver/v4/neo4j"
+)
+
+func (ec EvalContext) evalNeo4j(panel *PanelInfo, dbInfo DatabaseConnectorInfoDatabase, server *ServerInfo, w io.Writer) error {
+	_, conn, err := ec.getConnectionString(dbInfo)
+	if err != nil {
+		return err
+	}
+
+	password, err := ec.decrypt(&dbInfo.Password)
+	if err != nil {
+		return err
+	}
+
+	driver, err := neo4j.NewDriver(conn, neo4j.BasicAuth(dbInfo.Username, password, ""))
```

Comment:
Is username and password always required? If not then this auth bit should probably be conditional

Reply:
it is required by neo4j